### ISSUE TITLE
Issue with tree shaking

### ISSUE THREAD

**Issue by xyven1** [tommyip/bootstrap-icons-vue#1]
I am having trouble getting tree shaking working. I import the two icons I am using
```js
import { createApp } from 'vue'
import App from './App.vue'
import './registerServiceWorker'
import { io } from 'socket.io-client'
import {BIconList, BIconMap} from 'bootstrap-icons-vue'

const app = createApp(App)

app.config.globalProperties.$socket = io(process.env.VUE_APP_URL)
app.component('BIconList', BIconList)
app.component('BIconMap', BIconMap)

app.mount('#app')
```
But webpack-bundle-analyzer shows that practically the entire lib.esm.js loaded. 
![image](https://user-images.githubusercontent.com/35360746/103242583-b5a3e700-4924-11eb-9b47-3fcd02ad030c.png)
![image](https://user-images.githubusercontent.com/35360746/103242629-e6841c00-4924-11eb-89c0-0d033188286c.png)

**Comment by tommyip** [comment]
It appears that Webpack/Terser is more conservative than Rollup in terms of dead code elimination. It failed to detect that the [`genIcon`](https://github.com/tommyip/bootstrap-icons-vue/blob/master/src/utils.ts#L12) function is pure, so all the icons are bundled even if they aren't used. Adding `/*#__PURE__*/` annotations to each invocation of the function fixes the problem. I'll push a patch soon.

Thanks for the report @Xyven1.

**Comment by tommyip** [comment]
Released [`v0.4.1`](https://github.com/tommyip/bootstrap-icons-vue/releases/tag/v0.4.1).

465B gzip when importing only two icons. If there are still issues please let me know.
![bootstrap-icons-vue-filesize](https://user-images.githubusercontent.com/18504232/103435466-c7013200-4c49-11eb-8fb1-c86308094030.png)

**Comment by xyven1** [comment]
Thanks for the quick response. Webpack is now correctly shaking the package.
